config/locales/server.de.yml

```diff
@@ -1363,6 +1363,7 @@ de:
     log_mail_processing_failures: "Protokolliere Fehler bei der E-Mail-Verarbeitung in http://yoursitename.com/logs"
     email_in: "Erlaube Benutzern, neue Themen per E-Mail einzutragen (erfordert manuelles oder POP3-Polling). Konfiguriere die Adressen im „Einstellungen“-Reiter in jeder Kategorie."
     email_in_min_trust: "Minimale Vertrauensstufe um neue Themen per E-Mail erstellen zu können."
+    email_in_spam_header: "Der E-Mail-Header für die Spam-Erkennung."
```

Comment:
Quick note: this will be overwritten once we retrieve the translations from transifex. It's best to only provide English strings and then translate them on transifex.